src/sentry/profiles/flamegraph.py

```diff
@@ -0,0 +1,36 @@
+from typing import Dict, List, Optional
+
+from snuba_sdk import Column, Condition, Op
+
+from sentry.search.events.builder import QueryBuilder
+from sentry.search.events.types import ParamsType
+from sentry.snuba.dataset import Dataset
+from sentry.snuba.referrer import Referrer
+from sentry.utils.snuba import raw_snql_query
+
+
+def get_profiles_id(
+    params: ParamsType,
+    query: Optional[str] = None,
+) -> Dict[str, List[str]]:
+    builder = QueryBuilder(
+        dataset=Dataset.Discover,
+        params=params,
+        query=query,
+        selected_columns=["profile.id"],
+        limit=100,
+    )
```

Comment:
Previously in vroom I was actually sorting and taking the latest 100 profiles.

Here I've purposely removed the sorting to avoid that. The idea is: when the user set the date range, take a sample of 100 within that range, without biasing the result toward the most recent ones.

It could very much end up picking the latest inserted (not sure how clickhouse would choose if no constraint are applied) but I'd like to avoid setting that constraint of purpose.